### PR TITLE
Fixed #161 - Mentioned clearly that django-formtools is Python 3 only

### LINKED ISSUE #161: [BUG]: Release 2.2 breaking Python 2.7
The release from 2h ago is breaking python 2.7 (in my case tests).
It is clear that you wanted to deprecate the support for python 2.7 . 
But for pip to not allow the installation on python 2.7,
https://github.com/jazzband/django-formtools/blob/bffb403ceaca78563548dd01d1103373f12c80ac/setup.py#L135
needs to be:
`"Programming Language :: Python :: 3 :: Only",`

`"Programming Language :: Python` referes to universal python (2 __and__ 3), which caused PyPi to allow the installation of 'django-formtools==2.2' on python 2.7.

Same as 
https://github.com/jazzband/django-formtools/blob/bffb403ceaca78563548dd01d1103373f12c80ac/setup.cfg#L17
which also should be deleted to prevent mixups, also the meta information in `setup.py` are more important for PyPi's 'match making' I think.

### PR DESCRIPTION
**Pull request by claudep**
Thanks Sebastian Weigand for the report.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -134,6 +134,7 @@ def find_package_data(where='.', package='',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
```
Comment (s-weigand) [inline setup.py]: This line (135: `'Programming Language :: Python',`) should be deleted, so PyPi won't think it's python 2 compatible.

Comment (claudep) [reply setup.py]: I copied the same syntax as Django itself, so this should be fine, I guess.

Comment (s-weigand) [reply setup.py]: Just tried it and you are right, the line with the `:: Only` is the game changer.

Comment (hramezani) [reply setup.py]: I don't think that we need to remove this line. at least we have something like this in Django:
https://github.com/django/django/blob/6410d38ca7255961c08f1d657fdf920555d113f3/setup.cfg#L17

Comment (s-weigand) [reply setup.py]: Thats what I ment, there is no need to delete that line. 
All fine I guess, so go for it 👍

**Pull request merged**